src/containers/pages/MDAPoint/ClientUpload/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,3811 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
This snapshot must be made MUCH smaller.  Please see https://kentcdodds.com/blog/effective-snapshot-testing/